### PR TITLE
Fix the Connection Wizard

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #321 ([Wizard] Document new Wizard in 3.0+)

Fixes the outdated connection wizard plus some additional account related things.

Backport to 4.0 and 4.1

### CODE REVIEW

File: modules/ROOT/pages/using.adoc [hunk 1]
```diff
@@ -77,53 +77,66 @@ If a directory includes ignored files that are marked with warning icons that do
...
+a| image::using/wizard-screen-grant-access.png[form for entering your ownCloud login, width=320,pdfwidth=60%]
+|===
 --
 
-* Select what and how to sync.
+* Advanced Configuration:
 +
 --
-On the _"Local Folder Option"_ screen you may sync all of your files on the ownCloud server or select individual folders. The default local sync folder is `ownCloud`, in your home directory. You may change this as well.
+When enabling btn:[Advanced configuration], you can refine your synchronisation configuration if desired. Here you can define the synchronisation method like using the xref:vfs.adoc[Virtual Filesystem] or the location for  synchronized data.
```
Comment (EParzefall) [inline modules/ROOT/pages/using.adoc:114]:
```suggestion
When enabling btn:[Advanced configuration], you can refine your synchronization configuration if desired. Here you can define the synchronization method like using the xref:vfs.adoc[Virtual Filesystem] or the location for  synchronized data.
```

File: modules/ROOT/pages/using.adoc [hunk 2]
```diff
@@ -77,53 +77,66 @@ If a directory includes ignored files that are marked with warning icons that do
...
+* Final step:
 +
 --
-After selecting your sync folders, click the _"Connect"_ button at the bottom right. The Desktop app will attempt to connect to your ownCloud server, and if it is successful you'll see two buttons:
-
-* One to connect to your ownCloud Web UI.
-* One to open your local folder.
-
-It will also start synchronizing your files.
+When fine with the configuration made, click the btn:[Finish] button at the bottom right. The Desktop app will attempt to connect to your ownCloud server, and will start synchronizing your files.
```
Comment (EParzefall) [inline modules/ROOT/pages/using.adoc:122]:
```suggestion
Once you are satisfied with the configuration made, click the btn:[Finish] button at the bottom right. The Desktop app will attempt to connect to your ownCloud server and will start synchronizing your files.
```

File: modules/ROOT/pages/using.adoc (hunk 1 above)
Comment (phil-davis) [inline modules/ROOT/pages/using.adoc:114]:
```suggestion
When enabling btn:[Advanced configuration], you can refine your synchronisation configuration if desired. Here you can define the synchronisation method like using the xref:vfs.adoc[Virtual Filesystem] or the location for synchronized data.
```

File: modules/ROOT/pages/using.adoc [hunk 3]
```diff
@@ -77,53 +77,66 @@ If a directory includes ignored files that are marked with warning icons that do
...
 --
 
 === Adding New Accounts
 
-You may configure multiple ownCloud accounts in your Desktop App. Simply click the menu:Account[Add New] button on any account tab to add a new account, and then follow the account creation wizard. The new account will appear as a new tab in the settings dialog, where you can adjust its settings at any time. Use menu:Account[Remove] to delete accounts.
+You may configure multiple ownCloud accounts in your Desktop App. Simply click the menu:Account[Add New] button in any already configured account and follow the xref:connection-wizard[Connection Wizard]. The new account will appear as a new item in the main window on top.
```
Comment (EParzefall) [inline modules/ROOT/pages/using.adoc:127]:
```suggestion
You may configure multiple ownCloud accounts in your Desktop app. Simply click the menu:Account[Add New] button in any already configured account and follow the xref:connection-wizard[Connection Wizard]. The new account will appear as a new item in the main window on top.
```

File: modules/ROOT/pages/using.adoc [hunk 4]
```diff
@@ -77,53 +77,66 @@ If a directory includes ignored files that are marked with warning icons that do
...
 
-[NOTE]
-====
-To use *Two-Factor Authentication* (2FA), ownCloud server must have the {oauth2-app-url}[OAuth2 app] installed, configured and enabled. Please contact your ownCloud administrator for more details.
-====
+=== Removing An Account
```
Comment (EParzefall) [inline modules/ROOT/pages/using.adoc:129]:
```suggestion
=== Removing an Account
```

File: modules/ROOT/pages/using.adoc [hunk 5]
```diff
@@ -77,53 +77,66 @@ If a directory includes ignored files that are marked with warning icons that do
...
+
+To remove an account, select the account to be removed and the button menu:Account[Remove] to remove it. Note that the account will be removed but no synchronisation data deleted.
```
Comment (phil-davis) [inline modules/ROOT/pages/using.adoc:131]:
```suggestion
To remove an account, select the account to be removed and the button menu:Account[Remove] to remove it. Note that the account will be removed but no synchronized data deleted.
```

Comment (EParzefall) [inline modules/ROOT/pages/using.adoc:131]:
```suggestion
To remove an account, select the account to be removed and click on the button menu:Account[Remove] to remove it. Note that the account will be removed but no synchronization data deleted.
```

**Review by phil-davis — APPROVED** [review]
Just small suggestions

File: modules/ROOT/pages/using.adoc [hunk 6]
```diff
@@ -77,53 +77,66 @@ If a directory includes ignored files that are marked with warning icons that do
...
+
+=== Configuring Common Settings
+
+The settings tab contains common settings available and valid for all accounts configured. Here you can define network settings, bandwidth limitations, a confirmation setting to limit folder synchronisation size and more.
```
Comment (EParzefall) [inline modules/ROOT/pages/using.adoc:135]:
```suggestion
The settings tab contains common settings available and valid for all accounts configured. Here you can define network settings, bandwidth limitations, a confirmation setting to limit folder synchronization size and more. 
```

File: modules/ROOT/pages/using.adoc [hunk 7]
```diff
@@ -77,53 +77,66 @@ If a directory includes ignored files that are marked with warning icons that do
...
 +
 --
 Note that the URL given to you can be a different one than the final URL used by the Desktop app. This can be the case if a company has a generic setup URL that provides the Desktop app information about where to connect you. This mechanism is called https://en.wikipedia.org/wiki/WebFinger[WebFinger] and has the big benefit that one only needs to know the generic setup URL. The Desktop app tries to identify if webfinger is provided via the given URL and acts accordingly based on the server's response. This all runs in the background but is explained here in case you see a different URL being used than entered.
 
-image::using/client-1.png[form for entering ownCloud instance URL, width=500,pdfwidth=60%]
+image::using/wizard-screen-url.png[form for entering ownCloud instance URL, width=500,pdfwidth=60%]
 --
 
-* Enter your ownCloud login on the next screen.
+* Enter your ownCloud login on the next screen. Depending on what the administrator has configured, you may see different login screens. Note that with OpenID Connect (OIDC), you will be redirected to a browser for authentication:
```
Comment (michaelstingl) [inline modules/ROOT/pages/using.adoc:90]:
```suggestion
* Enter your ownCloud login on the next screen. Depending on what the administrator has configured, you may see different login screens. Note that with OAuth 2.0 and with OpenID Connect (OIDC), you will be redirected to a browser for authentication:
```

File: modules/ROOT/pages/using.adoc [hunk 8]
```diff
@@ -77,53 +77,66 @@ If a directory includes ignored files that are marked with warning icons that do
...
+
+=== Configuring Common Settings
+
+The settings tab contains common settings available and valid for all accounts configured. Here you can define network settings, bandwidth limitations, a confirmation setting to limit folder synchronisation size and more. 
 
-=== Configuring ownCloud Account Settings
+=== Configuring Account Settings
 
-At the top of the window are tabs for each configured sync account, and three others for Activity, General and Network settings. On your account tabs you have the following features:
+At the top of the window are tabs for each configured synchronisation account, and tow others for Activity and Settings. Selecting one account, you have the following features:
```
Comment (EParzefall) [inline modules/ROOT/pages/using.adoc:139]:
```suggestion
At the top of the window are tabs for each configured synchronization account and two more for Activity and Settings. Selecting one account, you have the following features available:
```

**Review by EParzefall — APPROVED** [review]

File: modules/ROOT/pages/using.adoc [hunk 9]
```diff
@@ -77,53 +77,66 @@ If a directory includes ignored files that are marked with warning icons that do
...
+====
+To use *Two-Factor Authentication* (2FA) with the ownCloud Server backend, ownCloud Server must have the {oauth2-app-url}[OAuth2 app] installed, configured and enabled. Please contact your ownCloud administrator for more details.
+====
+
+[width=100%,cols="^.^50%,^.^50%"]
+|===
+2+| *Login with ownCloud embedded user database*
+2+a| image::using/wizard-screen-login.png[form for entering your ownCloud login, width=500,pdfwidth=60%]
+
+2+| *Login with OIDC* +
```
Comment (mmattel) [inline modules/ROOT/pages/using.adoc:103]:
```suggestion
2+| *Login with OAuth 2.0 or OIDC* +
```

File: modules/ROOT/pages/using.adoc (hunk 3 above)
Comment (michaelstingl) [inline modules/ROOT/pages/using.adoc:127]:
```suggestion
You may configure multiple ownCloud accounts in your Desktop App. Simply click the menu:Account[Add New] button on the top left and follow the xref:connection-wizard[Connection Wizard]. The new account will appear as a new item in the main window on top.
```

File: modules/ROOT/pages/using.adoc (hunk 8 above)
Comment (michaelstingl) [inline modules/ROOT/pages/using.adoc:139]:
```suggestion
At the top of the window are tabs for each configured synchronisation account, and two others for Activity and Settings. Selecting one account, you have the following features:
```

**Pull request merged**